src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -202,6 +150,33 @@ def main(self):
             default=[],
             help='Devices to provision OSDs',
         )
+        parser.add_argument(
```

Comment:
why was all this moved into `__init__` ? adding the parsing in the constructor makes it harder to test (there are no tests added with these changes)

Reply:
I think `__init__` is the right place for argument parsing, since putting it into `main` requires the run under `root` just to see the help text. As for testing, as I mentioned in a previous comment, I'll add testing once this is no longer RFC. As for test-ablity: I'm not sure we need to test what argparse does..?